### PR TITLE
July 12 Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 616f172 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-483.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/amplify_relnotes/20240712_amplify.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+
+#### Enhancements for Agent SDK
+
+There are no enhancements or bug fixes in this release.
+
+##### Bug fixes for Agent SDK
+
+| Case ID     | Internal ID  | Description                                       |
+|-------------|--------------|---------------------------------------------------|
+|             | APIGOV-28305 | **Issue**: An issue was found where the Kafka Discovery Agent would timeout with an error. <br />**Resolution**: A fix was made to the configuration of the Kafka Discovery Agent to prevent the timeout error. |
```
Comment (jcollins-axway) [inline content/en/docs/amplify_relnotes/20240712_amplify.md:63]: This is kafka agent specific, not sdk

Comment (jcollins-axway) [reply content/en/docs/amplify_relnotes/20240712_amplify.md:63]: I think APIGOV-28034 is what is meant to be here

Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/20240712_amplify.md]: made the edits

**Review by robertengaxway — APPROVED** [review]
approved

File: content/en/docs/amplify_relnotes/20240712_amplify.md [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+
+#### Enhancements for Agent SDK
+
+There are no enhancements or bug fixes in this release.
+
+##### Bug fixes for Agent SDK
+
+| Case ID     | Internal ID  | Description                                       |
+|-------------|--------------|---------------------------------------------------|
+|             | APIGOV-28034 | **Issue**: An issue was found with the Traceability Agents where in some cases the event handling cause a consumer to not be correlated correctly to a transaction. <br />**Resolution**: A fix was made to the event handling logic for these cases. |
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20240712_amplify.md:63]: A fix was made to properly correlate the consumer with the transaction.

File: content/en/docs/amplify_relnotes/20240712_amplify.md [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+
+All the on-premise agents were affected by changes to the Agent SDK.
+
+#### Axway AWS Gateway Agent
+
+The following enhancements and bug fixes are available in this release.
+
+##### Enhancements for AWS Gateway Agent
+
+* **AWS Polling Mode support** - The on-premise AWS Agents only support AWS polling mode and a cost savings with the removal of the AWS Simple Queue Service.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20240712_amplify.md:79]: have no idea what this AWS polling mode support is. Was it supposed to replace the AWS SQS which leads to cost savings?

File: content/en/docs/amplify_relnotes/20240712_amplify.md [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+
+#### Enhancements for SaaS (embedded) agents
+
+There are no enhancements in this release.
+
+#### Bug fixes for SaaS (embedded) agents
+
+| Case ID     | Internal ID  | Description                                       |
+|---------    |--------------|---------------------------------------------------|
+| 01610765    | APIGOV-28290 | **Issue**: The display of the name (title) of API resources discovered by the SaaS embedded Azure agent was incorrect. <br />**Resolution**: The name (title) of API services discovered by the SaaS embedded Aaure agent accurately. The name is also reflected in the Marketplace as well correctly. |
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20240712_amplify.md:123]: The API Service name (title) is now accurately represented in Service Registry and Marketplace.

File: content/en/docs/amplify_relnotes/20240712_amplify.md (hunk 2 above)
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/20240712_amplify.md:63]: fixed

File: content/en/docs/amplify_relnotes/20240712_amplify.md (hunk 4 above)
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/20240712_amplify.md:123]: fixed

File: content/en/docs/amplify_relnotes/20240712_amplify.md [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+
+#### Bug fixes for Marketplace
+
+| Case ID   | Internal ID  | Description                                       |
+|-----------|--------------|---------------------------------------------------|
+| 01591849  | APIGOV-27945 | **Issue**: Marketplace Access Request allowing multiple application with same name. <br />**Resolution**: A check is done when creating an application to prevent having the same application name using the same subscription. For each subscription, application name is now unique to that subscription. |
+| 01604288  | APIGOV-28233 | **Issue**: Access request screen is crashing. <br />**Resolution**: An error while reading the AssetRequestDefintion schema has been fixed to prevent the issue. |
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20240712_amplify.md:239]: An error has been fixed to prevent the issue.

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

File: content/en/docs/amplify_relnotes/20240712_amplify.md (hunk 3 above)
Comment (robertengaxway) [reply content/en/docs/amplify_relnotes/20240712_amplify.md:79]: We now poll Cloudwatch, there is no need for SQS which is why its removed

**Pull request merged**